stdlib/http/client.pyi

```diff
@@ -99,6 +99,23 @@ responses: dict[int, str]
 
 class HTTPMessage(email.message.Message):
     def getallmatchingheaders(self, name: str) -> list[str]: ...  # undocumented
+    # override below all of Message's methods that use `_HeaderType` / `_HeaderTypeParam` with `str`
+    # avoiding having to make Message generic
```

Comment:
I would express this differently. It's not so much about avoiding making `Message` generic, but about the fact that `HTTPMessage` breaks the LSP and only accepts `str`.

Reply:
```suggestion
    # override below all of Message's methods that use `_HeaderType` / `_HeaderTypeParam` with `str`
    # `HTTPMessage` breaks the Liskov substitution principle by only intending for `str` headers
    # This is easier than making `Message` generic
```